doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,248 @@
+| network_plane_infos | List | Yes | vlan_id,sub_network(CIDR),gateway(could be null)|
+
+<p align="center"><b>Table 5-3-5-1:</b> Network information.</p>
+
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| vlan_id | String | Yes | vlan id for current network |
+| sub_network | String | Yes | subnet in CIDR format.|
+| gateway | String | No | gateway(could be null)|
+
+<p align="center"><b>Table 5-3-5-2:</b> Network plane information.</p>
+
+
+
+### 5.3.6 Controller informations
+List of controller nodes that designed for current VIM deployment.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| device_name | String | Yes | must be the device_name defined in server_information(###5.3.3), which would be set on Server from BMC|
+| node_name | String | Yes | node name for the controller to deploy, e.g: controller01 |
+| az_name | String | Yes | availability zone name which this node belongs to |
+| ha_name | List | Yes | HA name which this node assigned to, could be more than one HA it belongs.|
+| manage_vlan_id | String | Yes | may multiple vlan_id for management plane. Setting the dedicated vlan id assigned.|
+| storage_vlan_id | String | Yes | may multiple vlan_id for storage plane. Setting the dedicated vlan id assigned.|
+
+<p align="center"><b>Table 5-3-6:</b> Controller information.</p>
+
+### 5.3.7 Compute informations
+List of compute nodes that designed for current VIM deployment.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| device_name | String | Yes | must be the device_name defined in server_information(###5.3.3), which would be set on Server from BMC|
+| node_name | String | Yes | node name for the compute to deploy, e.g: compute01 |
+| az_name | String | Yes | availability zone name which this node belongs to |
+| ha_name | List | Yes | HA name(s) which this node assigned to.|
+| manage_vlan_id | String | Yes | may multiple vlan_id for management plane. Setting the dedicated vlan id assigned.|
+| storage_vlan_id | String | Yes | may multiple vlan_id for storage plane. Setting the dedicated vlan id assigned.|
+| vnic_type | String | Yes | VIM verification needs know compute node type: OVS,DPDK_OVS or SR-IOV node.|
```

Comment:
Can you define types in different sections like vnic_type, network_plane_type etc.

Reply:
do you mean adding another attribute of network_plane_type at here? not sure the what's the usage of network_plane_type to be added here. For the vnic_type, we think it should be used during verification stage.